install/kubernetes/cilium/charts/hubble-relay/Chart.yaml

```diff
@@ -1,7 +1,23 @@
 apiVersion: v2
 name: hubble-relay
-version: 1.8.90
-appVersion: 1.8.90
-description: Helm chart for running Hubble Relay
-sources:
-  - https://github.com/cilium/cilium
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 1.7.9
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: 1.7.9
```

Comment:
nit: I think that for the pre-1.9 charts we typically version these as `1.8.90`.